app/middlewares/checkAuthentication.js

```diff
@@ -20,7 +21,11 @@ const decode = token => {
         }
 
         return models.session.findOne({ where: { userId: user.id } }).then(session => {
-          if (!session || session.id !== result.token || result.expireTime < new Date()) {
+          if (
+            !session ||
+            session.id !== result.token ||
+            result.expireTime < moment().format('MMMM Do YYYY, h:mm:ss a')
```

Comment:
Standard date format for the application should be a constant.
Also fyi: You can also do `moment(result.expireTime) < moment()` to have them both in the same format  for comparison.